README.md

```diff
@@ -1,3 +1,4 @@
+<!-- markdownlint-disable no-duplicate-heading -->
```

Comment:
How about setting `siblings_only` (https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md#md024---multiple-headings-with-the-same-content) instead? I think that's the spirit of what's happening in these docs right?

I am wondering why that's not enabled by default. Reading guides like https://www.tpgi.com/heading-off-confusion-when-do-headings-fail-wcag indicate it's not necessarily bad to have duplicate _non-sibling_ headings...

Reply:
> I am wondering why that's not enabled by default.

Definitely agree.  I've updated it to use the `siblings_only` config